pkg/cloudwatchmetrics/cloudwatchmetrics_test.go

```diff
@@ -23,7 +23,16 @@ var (
 
 const maxTries = 14
 
+// Run only if AWS credentials are available
+func hasNoValidAWSSession() bool {
+	return sess.Config.Region == nil || *sess.Config.Region == ""
```

Comment:
```suggestion
		if _, err := session.NewSession(); err != nil {
		        fmt.Println("Failed to create AWS session, skipping tests:", err)
				return false
		}
		return true
	}
```
Maybe a better way to check instead of Region